vite.config.mts

```diff
@@ -89,6 +92,14 @@ export default defineConfig({
         /* other options */
       },
     }),
+
+    //https://github.com/vitejs/vite/discussions/7915
+    legacy({
```

Comment:
@sirpy 
as far as I could read this should be the way to handle 'globalThis'
tested it and can now build and run both dev/preview from local